index.md

```diff
@@ -56,6 +60,10 @@ scala3-sections:
         description: "MOOCs to learn Scala, for beginners and experienced programmers."
         icon: "fa fa-cloud"
         link: /online-courses.html
+      - title: "Becoming a Scala OSS Contributor"
+        description: "How to start contributing to open-source Scala ecosystem"
+        icon: "fa fa-code-branch"
+        link: /scala3/newcomers/contributor.html
```

Comment:
that guide is specifically about contributing to scala itself, rather than a community project, so I think it would need a lot of reworking to "contextualise" it for general contributing

Reply:
I think the index page is more general than that, see e.g. https://docs.scala-lang.org/contribute/#tools-and-libraries which is specifically about community libraries.

Reply:
done these suggestions